distribution/server/src/assemble/bin.xml

```diff
@@ -125,6 +125,11 @@
         <exclude>org.apache.pulsar:pulsar-functions-api-examples</exclude>
         <!-- prevent adding any distribution .tar.gz files in lib -->
         <exclude>*:tar.gz</exclude>
+        <!-- prevent adding annotation libraries -->
```

Comment:
We should be able to remove the exclusion for lombok, since it shouldn't get picked up